ArduCopter/events.cpp

```diff
@@ -400,8 +400,9 @@ void Copter::set_mode_SmartRTL_or_land_with_pause(ModeReason reason)
         AP_Notify::events.failsafe_mode_change = 1;
         return;
     }
+    GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "SmartRTL Unavailable, Using Land Mode");
```

Comment:
These have to be outside the #endif  - the user has essentially chosen an invalid failsafe option, and we should definitely be warning them of that.